main.cpp

```diff
@@ -148,18 +193,52 @@ struct Bank
         float bankBalance = 4550.60f;
         int memberTime  = 600; //days
 
-        void cancelBankAccount(int customerNumber);
-        void talkEmployee(std::string question);
-        void robBank(int bankBalance = 0);
+        void cancelBankAccount();
+        std::string talkEmployee(std::string question);
+        void robBank();
     };
 
     float withdrawMoney(Customer customer, float moneyToWithdraw); //returns updated bank balance
     float depositMoney(Customer customer, float moneyToDeposit); //returns updated bank balance
-    float takeCredit(Customer customer, float amountCredit, int periodToRepay); //returns interest rate
+    float takeCredit(Customer customer, float amountCredit, float periodToRepay); //returns interest rate
 
-    Customer Peter;
+    Customer peter;
 };
 
+void Bank::Customer::cancelBankAccount()
+{
+    
+}
+
+std::string Bank::Customer::talkEmployee(std::string question)
+{
+    return question;
+}
+
+void Bank::Customer::robBank()
+{
+    if(bankBalance < 0)
+    {
+        
+    }
+}
+
+float Bank::withdrawMoney(Customer customer, float moneyToWithdraw)
+{
+    return customer.bankBalance -= moneyToWithdraw;
+}
+
+float Bank::depositMoney(Customer customer, float moneyToDeposit)
+{
+    return customer.bankBalance -= moneyToDeposit;
+}
```

Comment:
This type of code is extremely difficult to understand.
Are you assigning the customer balance and then returning customer balance? What exactly is being returned?
You should be focusing on writing simple easy to understand code.
These kinds of expressions are not easy to understand.

Separate the use of the assignment operator from the return statement.

- [ ] revise